content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -133,507 +157,595 @@ types available by default:
+mechanism that improves the financing conditions for small and medium-sized businesses, and allows
+them to increase their productivity, through the early collection of credits and receivables issued
+to their clients and/or vendors.
+
+For these transactions it's important to consider the following requirements:
 
-For these transactions it’s important to have into consideration the next requirements:
+- specific document types (201, 202, 206, etc)
+- the emisor should be eligible by the AFIP to MiPyME transactions
+- the amount should be bigger than 100,000 ARS
+- A bank account type CBU must be related to the emisor, otherwise the invoice can't
+  be validated, having these errors messages for example:
 
-- Specific document types (201, 202, 206, etc).
-- The emisor should be eligible by the AFIP to MiPyme transactions. 
-- The amount should be bigger than 100,000 ARS. 
-- A bank account type CBU must be related to the emisor, otherwise the invoice can’t 
-  be validated, having these errors messages for example:  
-  
-.. image:: argentina/argentina_edi_10.png
+.. image:: argentina/bank-account-relation-error.png
    :align: center
+   :alt: Bank account relation error.
 
-**Credit& Debit Notes:** When creating a Credit/Debit note related to a FCE document, it is 
-important take the next points into consideration:
+To set up the :guilabel:`Transmission Mode` go to settings and select one of either :guilabel:`SDC`
+or :guilabel:`ADC`.
+
+.. image:: argentina/transmission-mode.png
+   :align: center
+   :alt: Transmission Mode.
 
-- Use the Credit and Debit Note buttons, so the correct reference of the originator 
-  document passed to the note. 
+To change the :guilabel:`Transmission Mode` for a specific invoice, go to the
+:guilabel:`Other Info` tab and change it before confirming.
 
-.. image:: argentina/argentina_edi_11.png
+.. note::
+   Changing the :guilabel:`Transmission Mode` will not change the mode selected in
+   :guilabel:`Settings`.
+
+.. image:: argentina/transmission-mode-on-invoice.png
    :align: center
-   
-- The document letter should be the same than the originator document (either A or B).
-- The same currency as the source document must be used.  When using a secondary currency 
-  there is an exchange difference  if the currency rate is different between the emission day 
-  and the payment date, it’s possible to create a credit/debit note to decrease/increase the 
-  amount to pay in ARS. 
+   :alt: Transmission Mode on Invoice.
+
+When creating a :guilabel:`Credit/Debit` note related to a FCE document:
+
+- use the :guilabel:`Credit and Debit Note` buttons, so the correct reference of the originator
+  document passed to the note.
+- the document letter should be the same than the originator document (either A or B).
+- the same currency as the source document must be used. When using a secondary currency
+  there is an exchange difference  if the currency rate is different between the emission day
```

Comment:
Delete sneaky whitespace 😉 

This list btw is good example where we're right to use periods at the end of each list item since they are complete sentences that fill out the stem sentence that introduces them 🧠 
```suggestion
  there is an exchange difference if the currency rate is different between the emission day
```